lib/widgets/main_app_bar.dart

```diff
@@ -1,161 +1,150 @@
+    Key key,
+    this.scrollOffset = 0,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    OverlayScreen().saveScreens({
+      'display': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: <Widget>[
+            SizedBox(
+              height: 200,
+              width: double.infinity,
+              child: DecoratedBox(
+                decoration: BoxDecoration(
+                  color: Colors.white,
+                ),
+                child: Column(
+                  children: <Widget>[
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.assignment,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board One'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.assignment,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Two'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                    Card(
+                      child: ListTile(
+                          leading: Icon(
+                            Icons.backup,
+                            color: Theme.of(context).primaryColor,
+                            size: 30.0,
+                          ),
+                          title: Text('Board Three'),
+                          onTap: () => print('Board Clicked')),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      'edit_home': CustomOverlayScreen(
+        backgroundColor: Colors.transparent,
+        content: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            SizedBox(
+
+            ),
+          ],
+        ),
+      ),
+    });
+    return Container(
+      padding: const EdgeInsets.symmetric(
+        vertical: 10.0,
+        horizontal: 10.0,
+      ),
+      color: Theme.of(context).primaryColor,
+      child: Row(
+        children: [
+          Expanded(
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                // Readability Settings
+                GestureDetector(
+                  onTap: () async {
+                    OverlayScreen().show(
+                      context,
+                      identifier: 'edit_home',
+                    );
+                  },

```

Comment:
Instead of using `GestureDectector` here, I think @nganphan123 also have a `shared` component that functions as a `ListTile` - do you think we should use that instead? I'm not sure with how `OverlayScreen()` is being used here, but that might be helpful?!?